spec/support/contexts/search.rb

```diff
@@ -6,6 +6,12 @@ def search_for(query)
     click_button("Search")
   end
 
+  def new_search_for(query)
+    # Clear the query params.
+    visit(current_path)
+    search_for(query)
```

Comment:
Perhaps `visit(current_path)` should be the default instead of adding this new method? Since `fill_in("search")` no longer clears the entire search (only the advanced search text).

Reply:
I think it's fine like this for now. I wouldn't want to add another unnecessary page load, and implicitly checking can be fraught due to timing issues.